src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+	<div>
+		<form v-if="useFormSubmit" @submit.prevent="addToBasket" class="tw-w-full tw-flex">
+			<fieldset
+				class="tw-w-full tw-flex" :disabled="isAdding"
+				data-testid="bp-lend-cta-select-and-button"
+			>
+				<label
+					v-if="hideShowLendDropdown && !isLessThan25"
+					for="LoanAmountDropdown"
+					class="tw-sr-only"
+				>
+					Lend amount
+				</label>
+				<div class="amountDropdownWrapper">
+					<kv-ui-select
+						v-if="hideShowLendDropdown && !isLessThan25"
+						id="LoanAmountDropdown"
+						class="tw-min-w-12"
+						v-model="selectedOption"
+					>
+						<option
+							v-for="priceOption in prices"
+							:key="priceOption"
+							:value="priceOption"
+						>
+							${{ priceOption }}
+						</option>
+					</kv-ui-select>
+				</div>
+
+				<!-- Lend button -->
+				<div :class="{ 'lendButtonWrapper' : hideShowLendDropdown}">
+					<kv-ui-button
+						key="lendButton"
+						v-if="lendButtonVisibility && !isLessThan25"
+						class="tw-inline-flex tw-flex-1"
+						data-testid="bp-lend-cta-lend-button"
+						type="submit"
+					>
+						{{ ctaButtonText }}
+					</kv-ui-button>
+				</div>
+
+				<!-- Lend again/lent previously button -->
+				<kv-ui-button
+					key="lendAgainButton"
+					v-if="isLentTo && !isLessThan25"
+					class="tw-inline-flex tw-flex-1"
+					data-testid="bp-lend-cta-lend-again-button"
+					type="submit"
+				>
+					Lend again
+				</kv-ui-button>
+
+				<!-- Stranded loans -->
+				<lend-amount-button
+					class="tw-w-full"
+					:loan-id="loan.id"
+					:show-now="false"
+					:amount-left="unreservedAmount"
+					@add-to-basket="addToBasket"
+					v-if="isLendAmountButton"
+				/>
+
+				<!-- Adding to basket button -->
+				<kv-ui-button
+					v-if="isAdding"
+					class="tw-inline-flex tw-flex-1"
+				>
+					Adding to basket
+				</kv-ui-button>
+			</fieldset>
+		</form>
+
+		<!-- Continue to checkout button -->
+		<kv-ui-button
+			v-if="this.state === 'basketed'"
+			variant="secondary"
+			class="tw-inline-flex tw-flex-1"
+			data-testid="bp-lend-cta-checkout-button"
+			to="/basket"
+		>
+			Checkout now
+		</kv-ui-button>
+
+		<!-- Funded, refunded, expired/ allSharesReserved button -->
+		<kv-ui-button
+			v-if="showNonActionableLoanButton"
+			class="tw-inline-flex tw-flex-1"
+		>
+			{{ ctaButtonText }}
+		</kv-ui-button>
+	</div>
+</template>
+
+<script>
+import {
+	buildPriceArray,
+	isLessThan25,
+	isBetween25And50,
+	isBetween25And500
+} from '@/util/loanUtils';
+
+import LendAmountButton from '@/components/LoanCards/Buttons/LendAmountButton';
+
+import KvUiSelect from '~/@kiva/kv-components/vue/KvSelect';
+import KvUiButton from '~/@kiva/kv-components/vue/KvButton';
+
+export default {
+	name: 'LendCtaExp',
+	inject: ['apollo', 'cookieStore'],
```

Comment:
Are these mixins still needed? If not, please remove. If so, add back to the stories. I'm guessing we don't need these anymore, now that the query is in the parent, but please take a look.
![image](https://user-images.githubusercontent.com/16867161/216454533-be1b05f1-70ab-45f1-9858-f4edfa76a8d3.png)

Reply:
They are no more needed, ty!